packages/toolpad-app/src/toolpad/AppEditor/AppEditorShell.tsx

```diff
@@ -159,14 +169,37 @@ export default function AppEditorShell({ appId, ...props }: ToolpadShellProps) {
           >
             Preview
           </Button>
-          <Button
-            variant="outlined"
-            endIcon={<RocketLaunchIcon />}
-            color="primary"
-            onClick={handleCreateReleaseDialogOpen}
-          >
-            Deploy
-          </Button>
+          <ButtonGroup>
+            <Button
+              variant="outlined"
+              endIcon={<RocketLaunchIcon />}
+              size="small"
+              color="primary"
+              onClick={handleCreateReleaseDialogOpen}
+            >
+              Deploy
+            </Button>
+            {isDeployed ? (
+              <React.Fragment>
+                <Button size="small" {...buttonProps}>
+                  <ArrowDropDownIcon />
+                </Button>
+                <Menu
+                  {...menuProps}
+                  anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}
+                  transformOrigin={{ horizontal: 'right', vertical: 'top' }}
+                >
+                  {!release.error ? (
+                    <MenuItem component="a" href={`/deploy/${appId}`} target="_blank">
+                      Open current deployed version
+                    </MenuItem>
+                  ) : (
+                    <MenuItem>{release.error as string}</MenuItem>
```

Comment:
@Janpot I'm not sure if that's the correct way to deal with `error` rendering - if I leave it without `as` TS complains that no overload was found matching the `unknown` which is the type for `error` by default.

Reply:
We have a utility function for this:
```tsx
errorFrom(release.error).message
```